src/nuklear_font.c

```diff
@@ -376,1689 +17,33 @@ nk_rp_pack_rects(struct nk_rp_context *context, struct nk_rp_rect *rects, int nu
-                r->x = (nk_rp_coord)((int)r->x + (int)pad);
-                r->y = (nk_rp_coord)((int)r->y + (int)pad);
-                r->w = (nk_rp_coord)((int)r->w - (int)pad);
-                r->h = (nk_rp_coord)((int)r->h - (int)pad);
-
-                nk_tt_GetGlyphHMetrics(info, glyph, &advance, &lsb);
-                nk_tt_GetGlyphBitmapBox(info, glyph, scale * (float)spc->h_oversample,
-                        (scale * (float)spc->v_oversample), &x0,&y0,&x1,&y1);
-                nk_tt_MakeGlyphBitmapSubpixel(info, spc->pixels + r->x + r->y*spc->stride_in_bytes,
-                    (int)(r->w - spc->h_oversample+1), (int)(r->h - spc->v_oversample+1),
-                    spc->stride_in_bytes, scale * (float)spc->h_oversample,
-                    scale * (float)spc->v_oversample, 0,0, glyph, alloc);
-
-                if (spc->h_oversample > 1)
-                   nk_tt__h_prefilter(spc->pixels + r->x + r->y*spc->stride_in_bytes,
-                        r->w, r->h, spc->stride_in_bytes, (int)spc->h_oversample);
-
-                if (spc->v_oversample > 1)
-                   nk_tt__v_prefilter(spc->pixels + r->x + r->y*spc->stride_in_bytes,
-                        r->w, r->h, spc->stride_in_bytes, (int)spc->v_oversample);
-
-                bc->x0       = (nk_ushort)  r->x;
-                bc->y0       = (nk_ushort)  r->y;
-                bc->x1       = (nk_ushort) (r->x + r->w);
-                bc->y1       = (nk_ushort) (r->y + r->h);
-                bc->xadvance = scale * (float)advance;
-                bc->xoff     = (float)  x0 * recip_h + sub_x;
-                bc->yoff     = (float)  y0 * recip_v + sub_y;
-                bc->xoff2    = ((float)x0 + r->w) * recip_h + sub_x;
-                bc->yoff2    = ((float)y0 + r->h) * recip_v + sub_y;
-            } else {
-                return_value = 0; /* if any fail, report failure */
-            }
-            ++k;
-        }
-    }
-    /* restore original values */
-    spc->h_oversample = (unsigned int)old_h_over;
-    spc->v_oversample = (unsigned int)old_v_over;
-    return return_value;
-}
-NK_INTERN void
-nk_tt_GetPackedQuad(struct nk_tt_packedchar *chardata, int pw, int ph,
-    int char_index, float *xpos, float *ypos, struct nk_tt_aligned_quad *q,
-    int align_to_integer)
-{
-    float ipw = 1.0f / (float)pw, iph = 1.0f / (float)ph;
-    struct nk_tt_packedchar *b = (struct nk_tt_packedchar*)(chardata + char_index);
-    if (align_to_integer) {
-        int tx = nk_ifloorf((*xpos + b->xoff) + 0.5f);
-        int ty = nk_ifloorf((*ypos + b->yoff) + 0.5f);
-
-        float x = (float)tx;
-        float y = (float)ty;
-
-        q->x0 = x;
-        q->y0 = y;
-        q->x1 = x + b->xoff2 - b->xoff;
-        q->y1 = y + b->yoff2 - b->yoff;
-    } else {
-        q->x0 = *xpos + b->xoff;
-        q->y0 = *ypos + b->yoff;
-        q->x1 = *xpos + b->xoff2;
-        q->y1 = *ypos + b->yoff2;
-    }
-    q->s0 = b->x0 * ipw;
```

Comment:
With which fonts did you test this?

Reply:
Roboto-Regular, ProggyClean, and some random font I had.